packages/SwingSet/src/initializeSwingset.js

```diff
@@ -269,6 +270,23 @@ export async function initializeSwingset(
     config.devices = {};
   }
 
+  // Use ambient process.env only if caller did not specify.
+  const { env: { WORKER_TYPE } = process.env } = runtimeOptions;
+  const defaultManagerType = config.defaultManagerType || WORKER_TYPE;
```

Comment:
Should we maybe name this `SWINGSET_WORKER_TYPE`? I wouldn't want it to collide with some other tooling that's talking about workers.

Reply:
yes, that seems worthwhile.